demo/index.html

```diff
@@ -1,69 +0,0 @@
-<!doctype html>
```

Comment:
We need to preserve these and put then into a storybook maybe.
in the story book load the json ownerTree and set it as property to each instance of cosmoz-treenode.